contrib/layout.py

```diff
@@ -0,0 +1,139 @@
+#!/bin/env python3
```

Comment:
`/usr/bin/env` is more likely to be available even on distros that don't symlink `/usr` into `/`.
```suggestion
#!/usr/bin/env python3
```